.github/workflows/release.yml

```diff
@@ -53,6 +53,29 @@ jobs:
         echo "::set-output name=tag_patch::${TAG_PATCH}"
         echo "::set-output name=tag_special::${TAG_SPECIAL}"
 
+    - name: Validate commits and tag signatures
+      run: |
+        
+        # Import Matt Holt's key
+        curl 'https://github.com/mholt.gpg' | gpg --import
+
+        # first we verify the tag, which are only accepted if signed by Matt's key
+        git verify-tag "${{ steps.vars.outputs.version_tag }}" || exit 1
```

Comment:
Can we do like `echo "Bad tag" && exit 1`? Is that a thing? Just so we know where we exited since there's more than one exit point in this step

Reply:
🥰 I'm scared as usual lol I'm going to mess this up 😓 but I'll try this is such a great way of learning .. thank you so much you made me cry I'm sorry for all my mishaps 💕

Reply:
Done.